reviews/2022-Antrea-annual.md

```diff
@@ -0,0 +1,114 @@
+* Antrea is used on Windows Nodes in GKE (Google Kubernetes Engine) and Google
+  [Anthos](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/user-guide-for-windows-server-os-node-pools)
+  to enforce K8s NetworkPolicies.
+* Antrea is used in [Tanzu Community Edition](https://github.com/vmware-tanzu/community-edition).
+* The Antrea community is running a weekly live show on our Youtube channel to explore different
+  topics related to K8s and to container networking: [Antrea
+  LIVE](https://www.youtube.com/watch?v=vJ49_BniUwI&list=PLuzde2hYeDBfHDD0zMbmG4QoVaSbkJChZ).
+* The community has added many significant features over the last year. One notable example is
+  "multi-cluster" support: the ability to connect different K8s clusters running Antrea to provide
+  network connectivity, policy enforcement, and service access.
+* We started the [Theia](https://github.com/antrea-io/theia) sub-project, an observability (flow
+  visibility) solution for Antrea.
+* The [website](https://antrea.io) is now automatically updated (by a bot) with each new release, to
+  ensure that the user-facing documentation is always up-to-date.
+
+## Metrics
+
+The project's devstats page can be found [here](https://antrea.devstats.cncf.io/).
+The numbers of contributions and contributors have remained consistent over the last year, and even
+show a slight upward trend since the beginning of 2022.
+The project has just reached 100 individual contributors (commits) on Github, even though a
+significant number of them only have one or two commits. In the last year, we have had contributions
+from different companies, including Intel, Google, and Transwarp Technology.
+The "PR Time to Engagement" has also been consistent, with an average of 8 hours over the last year.
+
+## Maintainers
+
+The [MAINTAINERS](https://github.com/antrea-io/antrea/blob/main/MAINTAINERS.md) file is up-to-date,
+and shows 4 current maintainers, all from the same company (VMware):
+
+```
+| Maintainer | GitHub ID | Affiliation |
+| ---------- | --------- | ----------- |
+| Antonin Bas | antoninbas | VMware |
+| Jianjun Shen | jianjuns | VMware |
+| Quan Tian | tnqn | VMware |
+| Salvatore Orlando | salv-orlando | VMware |
+```
+
+## Adoption
+
+We have an [ADOPTERS](https://github.com/antrea-io/antrea/blob/main/ADOPTERS.md) file in our
+repository, which shows 3 users: Glasnostic, Transwarp, TerraSky. These are users who have been
+active in the Antrea community and who have agreed to add their company name and logo to the file.
+
+The 2021 CNCF survey shows that among 518 respondents, 29 indicate that their company/organization
+uses Antrea in production, and 53 indicate that they are evaluating Antrea. A significant portion of
+these users are based in Asia. The survey data is available [on
+Github](https://github.com/cncf/surveys/blob/main/cloudnative/Cloud_Native_Survey_2021-Part_2.xlsx).
+
+While these are not "end users", we are also aware of the following usage of Antrea:
```

Comment:
Question for ToC: according to the updated adopter definition in [toc/FAQ.md](https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter), we should be able to add Google and VMware as "Service Provider" adopters, as they package and user Antrea in their services (GKE and VMware Tanzu on VMware Cloud), and Tanzu Community Edition and AKS Engine "Project" adopters. Could you confirm this? Does this help for Antrea to be considered for incubation?

Reply:
This is just for annual reviews, Incubation is a separate process.

Reply:
Understood. But I heard we can get feedback about incubation from the annual review process?